pkg/util/certificates.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2023 The KEDA Authors
```

Comment:
What about some unit tests for this file? Is it something that could be done? 🙏

Reply:
I can do it with a bit hacky solution

Reply:
done